WordPress/Classes/Models/KeyringConnection.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/**
+ KeyringConnection represents a keyring connected to a particular external service.
+ We only rarely need keyring data and we don't really need to persist it. For these
+ reasons KeyringConnection is treated like a model, even though it is not an NSManagedObject,
+ but also treated like it is a Remote Object.
+*/
+@objc public class KeyringConnection : NSObject
+{
+    public var additionalExternalUsers = NSArray()
+    public var dateIssued = NSDate()
+    public var dateExpires:NSDate? = nil
+    public var externalID = "" // Some services uses strings for their IDs
+    public var externalName = ""
+    public var externalDisplay = ""
+    public var externalProfilePicture = ""
+    public var label = ""
+    public var keyringID:NSNumber = 0
```

Comment:
Strictly speaking, probably not, but I think its a good practice.  Swifts bridging lets an Int be assigned to a NSNumber but not the other way around.  This saves some calls to `integerValue`.  It also keeps parity with the PublicizeConnection model which this object closely resembles.